lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -34,6 +34,11 @@ FAILED_RFC:
     next:
       targetState: NO_MATCH_PAGE_RFC
 
+FAILED_UPDATE_ADDRESS:
+  events:
+    next:
+      targetState: NO_MATCH_PAGE_ADDRESS
```

Comment:
Typo - this doesn't match the state below
`NO_MATCH_PAGE_ADDRESS`
vs
`NO_MATCH_PAGE_UPDATE_ADDRESS`